src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisInputBuilderService.java

```diff
@@ -78,8 +97,26 @@ private double getLoadWeight(Load load, SensitivityAnalysisInputData.Distributio
         }
     }
 
-    private Stream<IdentifiableAttributes> getIdentifiablesFromFilter(SensitivityAnalysisRunContext context, FilterIdent filter, List<IdentifiableType> equipmentsTypesAllowed, Reporter reporter) {
-        List<IdentifiableAttributes> listIdentAttributes = filterService.getIdentifiablesFromFilter(filter.getId(), context.getNetworkUuid(), context.getVariantId());
+    private List<IdentifiableAttributes> goGetIdentifiables(FilterIdent filter, UUID networkUuid, String variantId, Reporter reporter) {
+        try {
+            return filterService.getIdentifiablesFromFilter(filter.getId(), networkUuid, variantId);
+        } catch (Exception ex) {
+            LOGGER.error("Could not get identifiables from filter " + filter.getName(), ex);
+            reporter.report(Report.builder()
+                .withKey("sensitivityInputParametersTranslationFailure")
+                .withDefaultMessage("Could not get identifiables from filter " + filter.getName()) // as we do not yet show values
+                .withSeverity(TypedValue.ERROR_SEVERITY)
+                .withValue("exception", ex.getMessage())
+                .withValue("name", filter.getName())
```

Comment:
name and exception should appear in the default message with '${name}' and '${exception}'